src/components/bootstrap-overrides.css

```diff
@@ -82,3 +85,16 @@
   line-height: 16px;
   color: #ffffff;
 }
+
+.story-block-text {
+font-family: Roboto, Helvetica, sans-serif;
+font-style: normal;
+font-weight: bold;
```

Comment:
by default I don't think it makes sense to have a heavier font weight. this is may be appropriate for the 'big text' sections, but not for the our vision section.

Reply:
Removed it.

Reply:
I removed bold, but I tried setting the weight lower to no avail.